akka-stream-testkit/src/test/resources/reference.conf

```diff
@@ -16,6 +16,10 @@ akka.test.stream-dispatcher {
   mailbox-requirement = "akka.dispatch.UnboundedMessageQueueSemantics"
 }
 
+akka.test.stream {
+  timeout = 5 s
+}
```

Comment:
I think we need a better setting name, `akka.stream.testkit.all-stages-stopped-timeout`, also, it should go in a ´src/main/resources/reference.conf` since this conf won't be included in the published artifact.

Reply:
Done. Thanks for the feedback!